bundle/jsky.app.ot/src/main/java/jsky/app/jskycat/JSkyCat.java

```diff
@@ -405,28 +395,35 @@ public static void main(String args[]) {
         boolean ok = true;
         int tilecache = 64;
 
+        label:
```

Comment:
I assume it was used for testing JskyCat back in the day, it could be well removed